doc/architecture/decisions/0011-control-applicant-access-using-delius-team-caseload.md

```diff
@@ -0,0 +1,168 @@
+# 11. Control "applicant" access using Delius team caseload
+
+Date: 2023-01-23
+
+## Status
+
+Accepted
+
+## Context
+
+The service will manage all but one of its roles internally. This ADR concerns that one
+exception: the "applicant" role which is not modelled explicitly within the service, but
+is derived from caseload information from Delius.
+
+The Approved Premises service exposes personal data from NOMIS and from OASys and so
+it's necessary to restrict creating and viewing "applications" to probation officers with an
+appropriate caseload.
+
+
+
+### Overview of the service's roles
```

Comment:
Should this be part of the decision?

Reply:
hmm, I thought it was helpful to include in the "Context" section as background, but on reflection I think I agree that "less is more" and that it's context enough to say:

- we've got some internally managed rolew
- we've got one special default role whose capabilities are defined by team caseloads managed within Delius